### ISSUE TITLE
How to set the maxdisp value for a custom dataset?

### ISSUE THREAD

**Issue by hillaric** [cogsys-tuebingen/mobilestereonet#16]
Does the maxdisp value has a influence on the result of inference time or the performance? Besides, the result does not work well on my own dataset, I guess it is the problem of maxdisp value.

**Comment by samuel-woerz** [comment]
Yes, different maximum disparity demands of your data could be the reason for bad performance. 
Did you fine-tune the model on your data? This is often necessary for good performance, especially on data that does not closely resemble the traffic scenes of KITTI or DrivingStereo.
